app/services/database_manager.py

```diff
@@ -0,0 +1,231 @@
+from qdrant_client import QdrantClient
+from qdrant_client.http import models as rest
+from sentence_transformers import SentenceTransformer
+import fitz
+
+# Configuration for Qdrant client
+client = QdrantClient(host="localhost", port=6333)
+
+# Sentence-BERT model
+model = SentenceTransformer('sentence-transformers/all-MiniLM-L6-v2')
+
+
+def partition_text(text, chunk_size=100):
```

Comment:
Use type hints for function definitions, this will help you a lot to detect issues directly with the python linter. 

```python
def partition_text(text: str, chunk_size : int=100):
```